src/css/components/_modals.scss

```diff
@@ -127,6 +226,53 @@
         }
       }
     }
+    .add__member {
+      .add__member__form {
+        .email__header {
+          font-family: Inter;
```

Comment:
Inter is common throughout the app?

Reply:
No, it's not common but used frequently in Figma design